src/routes/+error.svelte

```diff
@@ -14,7 +14,7 @@
 <section title={$page.error?.message}>
   <em>404</em>
   <h1>Frank can't find where you're going!</h1>
-  <Button text={'Return to Home'} link={'/'} />
+  <Button text={'Return to Home'} link={'/'} redirect={'_self'} />
```

Comment:
maybe change the behavior of this such that the prop is redirect=false to prevent opening in a new tab? imo better abstraction from the raw html

Reply:
also i think default behavior should be to not redirect to new tab, there's only 3 files this button is imported in, can that be changed?

![image](https://user-images.githubusercontent.com/8981287/202880327-85da8185-45c6-4a4e-8a35-0e557cf60cf8.png)

Reply:
Got it, thanks for the suggestions. I just updated the PR so that the tweaks satisfy your requests. Let me know if anything else should be changed.